pkgs/development/python-modules/dependency-groups/default.nix

```
@@ -0,0 +1,42 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+  lib,
+  packaging,
+  pytestCheckHook,
+  pythonOlder,
+  tomli,
+}:
+
+buildPythonPackage rec {
+  pname = "dependency-groups";
+  version = "1.3.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "pypa";
+    repo = pname;
+    rev = version;
+    hash = "sha256-suuSx3zf0Y45FJdH8Cb6N7hcvPnzleREpHhtdiG2CLg=";
+  };
+
+  build-system = [ flit-core ];
+
+  dependencies = [
+    packaging
+  ] ++ lib.optional (pythonOlder "3.11") tomli;
+
+  nativeCheckInputs = [ pytestCheckHook ];
+  pythonImportsCheck = [ "dependency_groups" ];
+
+  meta = {
+    description = "A standalone implementation of PEP 735 Dependency Groups";
```

```suggestion
    description = "Standalone implementation of PEP 735 Dependency Groups";
```
Not start with the definite or an indefinite article.

---

now that you mention it, I saw the treewide PR about removing indefinite articles earlier. This is the "official" project description (from their github repo) though; does the rule still apply?